### PR TITLE
[EDU-3763] feat: guide for investigation flow with graphql

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new guide for investigation steps using GraphQL API.

[EDU-3763]: https://aziontech.atlassian.net/browse/EDU-3763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+To have your network list running, you need to go to **Edge Firewall** > **Rules Engine** and create a rule associated with the network list you've just created.
+
+<Button href="/en/documentation/products/guides/blocklists-ip-addresses-edge/#associate-network-list-with-edge-firewall-rule-set" text="go to associate network list to rule guide" variant="secondary" target="_blank"></Button>
+
+After the propagation time for the rule, your domain, associated with the edge firewall you added the list to, will block all requests from that country, which in this case, is Brazil.
+
+### Monitoring on playground again
+
+After creating the rule on **Rules Engine for Edge Firewall**, you need to continue monitoring your application traffic to guarantee the rule has been applied and the application traffic is normalized.
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:131]: After creating the rule in the Rules Engine for Edge Firewall

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+import Button from '~/components/Button.astro'
+
+Constant monitoring can help you identify unusual behaviors in your applications and possible attacks. For this guide, you'll use an example scenario where there are requests coming from Brazil, a country you don't want to allow access to, and you want to filter requests from Chile, the country most of your safe requests come from.
+
+:::tip
+Change the countries in this example to the ones that best fit your application and investigation scenario.
+:::
+
+See the steps described in this guide to perform an investigation on requests and next steps to resolve the situation it if you're receiving an attack.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:18]:
```suggestion
See the steps described in this guide to investigate requests and the next steps to resolve the situation if you're being attacked.
```

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+1. In Real-Time Events GraphQL API [playground](https://manager.azion.com/events/graphql), add the following query:
+
+```graphql
+query requestsInvestigationCountries {
+  httpEvents(
+    limit: 100,
+    filter: {
+      tsRange: {begin:"2024-03-20T10:10:10", end:"2024-03-27T10:10:10"}
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:40]:
```suggestion
      tsRange: {begin:"2024-03-20T10:10:10", end:"2024-03-27T10:10:10"
```

Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:40]:
```suggestion
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:40]: não tenho certeza se o espaço foi intencional

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+        "requestUri": "/directory/file.html",
+        "status": 403,
+        "count": 9564
+      }
+      ]
+  }
+}
+```
+
+3. Analyze the query response and check if there's incoming requests from the countries you don't want to allow, in this case, from Brazil.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:105]:
```suggestion
3. Analyze the query response and check if there are incoming requests from the countries you don't want to allow. In this case, from Brazil.
```

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+2. Click **+ Network List**.
+3. Give your network list a unique and easy-to-remember name.
+4. In **Type**, choose **Countries**.
+5. In **Countries**, select the countries you want to add to the list. In this case, **Brazil**.
+
+To have your network list running, you need to go to **Edge Firewall** > **Rules Engine** and create a rule associated with the network list you've just created.
+
+<Button href="/en/documentation/products/guides/blocklists-ip-addresses-edge/#associate-network-list-with-edge-firewall-rule-set" text="go to associate network list to rule guide" variant="secondary" target="_blank"></Button>
+
+After the propagation time for the rule, your domain, associated with the edge firewall you added the list to, will block all requests from that country, which in this case, is Brazil.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:127]:
```suggestion
After the propagation time for the rule, your domain, associated with the edge firewall you added the list to, will block all requests from that country. In this case, Brazil.
```

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+query requestsInvestigationStatus {
+  httpEvents(
+    limit: 100,
+    filter: {
+      tsRange: {begin:"2024-03-20T10:10:10", end:"2024-03-27T10:10:10"}
+
+      geolocCountryNameNotIn: ["Chile"]
+      hostIn: ["myhost.net", "anotherhost.com"]
+      statusIn: "403"
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/investigate-requests.mdx:147]:
```suggestion
same
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**